cypress/integration/entitybrowser_spec.js

```diff
@@ -62,17 +91,98 @@ describe('Entity Browser', function() {
 
     it('should change page back to list view without warning', function() {
       cy.get('[data-cy=entity-detail_back-button]')
-        .click({force: true})
+        .click()
+      cy.get('[data-cy=list-cell]', {timeout: 15000})
+        .should('have.length.above', 1)
+    })
+
+    it('should change page back to list view without warning if data did not change', function() {
+      cy.get('input#input-detailForm-firstname')
+        .type('{selectall}{del}Cypress Test')
+      cy.wait(200) // Wait for debounced onChange
+      cy.get('[data-cy=entity-detail_back-button]')
+        .click()
       cy.get('[data-cy=list-cell]', {timeout: 15000})
         .should('have.length.above', 1)
     })
 
     it('should display warning on changing page back to list view on edited form', function() {
       cy.get('input#input-detailForm-callname')
-        .type('{selectall}{del}')
+        .type('{selectall}{del}Test')
       cy.wait(200) // Wait for debounced onChange
       cy.get('[data-cy=entity-detail_back-button]').click()
       cy.contains('ungespeicherte Änderungen')
     })
+
+    it('should allow update of various fields', function() {
+      cy.get('#input-detailForm-firstname').type('!')
+      cy.get('#input-detailForm-lastname').type('!')
+      cy.get('#input-detailForm-email').type('test@tocco.ch')
+      cy.get('#input-detailForm-callname').type('cyp')
+      cy.get('#input-detailForm-relGender').type('{downarrow}').get('#react-select-3-option-1').click()
```

Comment:
`#react-select-3-option-1` looks like it could lead to problems someday - I guess we don't control the id of the option?

Reply:
you are right but for the moment i dont have a better solution